hawkbit-sdk/hawkbit-sdk-dmf/src/main/java/org/eclipse/hawkbit/sdk/dmf/DmfController.java

```diff
@@ -83,10 +85,24 @@ public void stop() {
     }
 
     public void processUpdate(final EventTopic actionType, final DmfDownloadAndUpdateRequest updateRequest) {
-        updateHandler.getUpdateProcessor(this, actionType, updateRequest);
+        log.info("Processing update for {} actionId : {}", controllerId, updateRequest.getActionId());
+        updateHandler.getUpdateProcessor(this, actionType, updateRequest).run();
+        updateRequest.getSoftwareModules().forEach(sm -> {
+            updateAttributes(sm.getModuleId().toString(), sm.getModuleVersion());
+        });
     }
 
     public void sendFeedback(final UpdateStatus updateStatus) {
+        log.info("Sending {} for controller : {} action : {}", updateStatus.status(), controllerId, currentActionId);
```

Comment:
maybe "Sending UPDATE_ACTION_STATUS ..." will be more inline with the log message of sendUpdateAttributes